context-free.md

```diff
@@ -2,21 +2,32 @@
 
 ##  Who is this programming language for?
 
+Context Free Art is a programing language built for people that want to design compture generated art peices. This can be anything from graphic art to desgins. I would say that the primary focus for this language is for those who want a simple way to design art online. This being simpler then Python or Java.
 
 ## What is easy to do in this language? Why is it easy?
 
+It was easy to make shapes in this language becuase all you need is the name of the shape you are trying to draw. It was also fairly easy to understand what the program was doing. For instance you start your file with startshape and you give it a variable name. After this all you have to do is define a rule for that varible such as draw or TRIANGLE. 
```

Comment:
I think the point about "all you need is the name of the shape you are trying to draw" is articulated in an interesting way. I wouldn't have thought of it that way, but even if you need to define a shape first, moving forward the user can use that shape just with the name. It feels more like building blocks to make a pictures, than math or algorithms.